PowerToys.sln

```diff
@@ -86,80 +86,202 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "PowerRenameUWPUI", "src\mod
 		{0E072714-D127-460B-AFAD-B4C40B412798} = {0E072714-D127-460B-AFAD-B4C40B412798}
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "imageresizer", "imageresizer", "{6C7F47CC-2151-44A3-A546-41C70025132C}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ImageResizer", "src\modules\imageresizer\src\ImageResizer\ImageResizer.csproj", "{2BE46397-4DFA-414C-9BD4-41E4BBF8CB34}"

```

Comment:
`src\modules\imageresizer\src\ImageResizer\`

This seems like an uneccesarily long source path.  Can we just shorten the path to src\modules\ImageResizer?

Reply:
The folder structure was like that since I directly just shifted the folder without any other changes. I wont be able to have it as src\modules\ImageResizer as there are 3 separate projects but I can shorten similar to what has been done for powerrename, i.e. src\modules\powerrename\dll, src\modules\powerrename\ui, etc

Reply:
@ryanbodrug-microsoft  I've fixed the folder structure and removed the setup and build files. The only other remaining file in this folder apart from the source code is the License file. Let me know if any other changes are needed.